src/CoreFoundation/CFMutableString.cs

```diff
@@ -140,7 +142,7 @@ public bool Transform (NSString transform, bool reverse)
 
 		public bool Transform (string transform, bool reverse)
 		{
-			var t = NSString.CreateNative (transform);
+			var t = CreateNative (transform);
```

Comment:
Context, NSString.CreateNative (...) is obsolete and the recommendation was to use CFString.CreateNative (...)